sycl/test/on-device/extensions/usm/usm_allocator_container.cpp

```diff
@@ -0,0 +1,60 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t1.out
+// RUN: %RUN_ON_HOST %t1.out
```

Comment:
```suggestion
// RUN: %RUN_ON_HOST %t1.out
// RUN: %CPU_RUN_PLACEHOLDER %t1.out
// RUN: %GPU_RUN_PLACEHOLDER %t1.out
// RUN: %ACC_RUN_PLACEHOLDER %t1.out
```
I suggest to execute on all supported devices

Reply:
I have added these.